cmd/config/config.go

```diff
@@ -64,9 +64,10 @@ func run() func(*cobra.Command, []string) error {
 				return err
 			}
 
-			output, err := output.CmdOutput(
+			output, err := output.CmdOutputResource(
```

Comment:
The output API changed to
1. split into two functions: one for a singular resource response and one for multiple resources
2. pass in the outputter function from the caller because there are as of now five different ways we can get a response to format

We could further refactor by creating a function for each possible combination of singular/multiple response + type of response.